pkg/config/config.go

```diff
@@ -19,6 +20,8 @@ type Config struct {
 	UPSImageStreamInitialImage        string
 	PostgresImageStreamInitialImage   string
 	OauthProxyImageStreamInitialImage string
+
+	BackupImage string
```

Comment:
Not using an imagestream for this one, since it's only used in a CronJob rather than a DeploymentConfig.